src/com/hms/flexyosisoftconnector/Tag.java

```diff
@@ -120,6 +193,417 @@ public void recordTagValue(String time)
       }
    }
 
+   /**
+    * exportBlockDescriptorHistoricalLogs
+    * Send an EBD command with a start and end time and a tag name.
+    * Start time is stored in an external file, end time is the start time

```

Comment:
"end time is the start time" - what does that mean?

Reply:
I meant the endtime is set to a ("start time" + "a time window")
I'll add a bit to this comment section